src/main/helm/repositories/add-helm-repository/add-helm-repository.injectable.ts

```diff
@@ -54,7 +55,18 @@ const addHelmRepositoryInjectable = getInjectable({
         args.push("--cert-file", certFile);
       }
 
-      return await execHelm(args);
+      const result = await execHelm(args);
+
+      if (result.callWasSuccessful) {
+        return {
+          callWasSuccessful: true,
+        };
+      }
+
+      return {
+        callWasSuccessful: false,
+        error: result.error.stderr || result.error.error.message,
```

Comment:
I'm seeing this `error.error` duplicated in many places. The funk is a bit too much. Improve naming? One could be "errorObject"? Or the root error could be "rejection"?

Reply:
Or maybe in this case we could just wrap it better.